src/SmartComponents/CompletedTaskDetailsKebab/CompletedTaskDetailsKebab.js

```diff
@@ -29,7 +30,7 @@ const CompletedTaskDetailsKebab = ({ status, setModalOpened }) => {
 
   useEffect(() => {
     setDropdownItems(createDropdownItems());
-  }, []);
+  }, [status]);
```

Comment:
I may be wrong, but I believe that dropdown items do not have to be recreated for certain statuses. If we give more explicit status, we would avoid re-rendering for these cases.